File: code/mathematical-algorithms/sum_of_digits/SumOfDigits.java
// Part of Cosmos by OpenGenus Foundation
public class SumOfDigits {
    public static int sumDigits(long n) {
        int sum = 0;
        while (n > 0) {
            sum += n % 10;
            n = n / 10;
        }
        return sum;
    }
}

File: algorithms/mathematical_algorithms/sum_of_digits/sum_of_digits.swift
//
//  sum_of_digits.swift
//  test
//
//  Created by Kajornsak Peerapathananont on 10/14/2560 BE.
//  Copyright © 2560 Kajornsak Peerapathananont. All rights reserved.
//
//Part of Cosmos by OpenGenus Foundation

import Foundation

func sum_of_digits(number : String)->Int{
    return number.characters.map({ char in
        return Int(String(char))!
    }).reduce(0, +)
   
}


